### PR TITLE
Homepage: Temporary button for US users linking to congress petition.

### PR DESCRIPTION
**Pull request by breville**
US visitors to code.org homepage should see the expandable petition replaced by a button going to the petition.

screenshot 2016-04-26 15 44 50

### CODE REVIEW

**Comment by breville** [comment]
fyi @aoby @philbogle @wjordan 

The ask was for this to appear for US visitors to the homepage, but I have a sneaking suspicion that using request.location.country_code will not work due to the CDN... is this the case?

**Comment by aoby** [comment]
I think you may be right. Do you know where request.location.country_code gets populated? Looks like we can configure Cloudfront to pass along the country but it's in a special header. See http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/header-caching.html#header-caching-web-location.

**Comment by wjordan** [comment]
iirc it should work through cloudfront, the real IP should be passed through the `x-forwarded-for` header and used by the geocoder. Have we tested this?

**Comment by breville** [comment]
A couple uses on the Rails side: https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=%22request.location%22&type=Code

**Comment by breville** [comment]
Switching to just use request.locale since we've used that a lot on Pegasus pages.  I don't think varnish was going to like the IP/geocoder-based approach.

File: pegasus/sites.v3/code.org/views/congress_link.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#petitionblock.greybox{style: "background-color:white; padding: 10px 0px; color: rgb(91,103,112)"}
+  %div{style: "clear:both"}
+  #petition_message{:style=>"font-size: 22px; line-height:28px; font-family: 'Gotham 5r', sans-serif;"}=I18n.t:petition_message
+  .container_responsive{style: "margin-top:10px"}
+    #threedivs{style: "text-align:center"}
+      %div#signpetitionclick{style: "display:inline-block; vertical-align:top; cursor: pointer; font-weight:bold; vertical-align: 5px;"}
+        %a{href: "https://bit.ly/computersciencepetition", target: "_blank"}
+          %button{ style: "margin-bottom:5px"}
+            Sign the Petition
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/congress_link.haml:9]: Nit, do we want to make this lower case petition?

**Comment by tanyaparker** [comment]
I think locale is sufficient. It's not like we don't allow non-US people to sign the petition. 

LGTM

**Pull request closed without merging**